frontend/app/views/fragments/form/marketingChoices.scala.html

```diff
@@ -1,12 +1,29 @@
-@(marketingChoices: Option[com.gu.identity.play.StatusFields])
+@(marketingChoicesOpt: Option[com.gu.identity.play.StatusFields])
+@marketingChoicesOpt match {
+    case Some(marketingChoices) => {
+        <fieldset class="u-h">
+            <input type="hidden"
+            name="marketingChoices.gnnMarketing"
+            value="@marketingChoices.receiveGnmMarketing" />
 
-<fieldset class="u-h">
+            <input type="hidden"
+            name="marketingChoices.thirdParty"
+            value="@marketingChoices.receive3rdPartyMarketing" />
+        </fieldset>
+    }
+    case None => {
+        <div class="fieldset__fields">
+
+            <div class="form-field js-billing-input-container">
+
+                <label class="label" for="use-delivery-address">
+                    <input type="checkbox" name="marketingChoices.gnnMarketing" id="receive-gnn-marketing" value="true"/>
```

Comment:
An HTML forms refresher (for me) - `value="true"` does not mean that the checkbox will default to checked, it just means that if it _is_ checked by the user, the value that will be sent encoded in the form data will be `true`.

https://www.w3.org/TR/html401/interact/forms.html#adef-value-INPUT